### ISSUE TITLE
CapabilityMap

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4102]
**Is your feature request related to a problem? Please describe.**
VIVO **CapabilityMap** should be aligned with AA level of WCAG.
